app/policies/dataservice/blob_policy.rb

```diff
@@ -0,0 +1,15 @@
+class Dataservice::BlobPolicy < ApplicationPolicy
+
+  def index?
+    admin?
+  end
+
+  def new_or_create?
+    admin?
```

Comment:
This is confusing because normal users need to be able to create and edit their own blobs.  I think it technically works because this code is currently bypassed when the blob is sent with a token.  A comment here would be good, so someone looking at this has a clue what is going on.